retrofit-adapters/rxjava2/src/main/java/retrofit2/adapter/rxjava2/CallExecuteObservable.java

```diff
@@ -61,6 +61,13 @@
     }
   }
 
+  Call<T> cloneCallIfAlreadyExecuted(Call<T> call) {
```

Comment:
This isn't worth a method–especially not a virtual one. Let's just do a ternary.
```java
Call<T> call = originalCall.isExecuted()
    ? originalCall.clone()
    : originalCall;
```